internal/gke/cluster_info.go

```diff
@@ -79,15 +79,15 @@ type ClusterInfo struct {
 // GetClusterInfo returns information about a GKE cluster running in a given
 // cloud region.
 // REQUIRES: The caller is running on the user machine.
-func GetClusterInfo(ctx context.Context, config CloudConfig, cluster, region string) (*ClusterInfo, error) {
+func GetClusterInfo(ctx context.Context, config CloudConfig, region string) (*ClusterInfo, error) {
 	// Fetch cluster credentials to the local machine.
 	kubeFileName := filepath.Join(
-		os.TempDir(), fmt.Sprintf("serviceweaver_%s_%s", cluster, uuid.New().String()))
+		os.TempDir(), fmt.Sprintf("serviceweaver_%s_%s", applicationClusterName, uuid.New().String()))
 	if _, err := runGcloud(config, "", cmdOptions{
 		EnvOverrides: []string{
 			fmt.Sprintf("KUBECONFIG=%s", kubeFileName),
 		}},
-		"container", "clusters", "get-credentials", cluster, "--region", region,
+		"container", "clusters", "get-credentials", applicationClusterName, "--region", region,
```

Comment:
Btw., will this still work if there's an existing config cluster? That cluster will be named `"serviceweaver_config"`, so I'm worried we won't be getting correct credentials for it.

Reply:
Great point. I added back the cluster name and set its value accordingly.